.prettierignore

```diff
@@ -0,0 +1,3 @@
+docs
+node_modules
+example/node_modules
```

Comment:
Test, however, I think the single `node_modules` on line 2 should also match it in any directory path. ie, line 3 shouldn't be needed.

Reply:
cool. will update